### PR TITLE
Use parse_bytes in read_bytes(sample=...)

### PR DESCRIPTION
**Pull request by mrocklin**
- [ ] Tests added / passed
- [ ] Passes `flake8 dask`

### CODE REVIEW

File: dask/bytes/tests/test_local.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def test_read_bytes():
 def test_read_bytes_sample_delimiter():
     with filetexts(files, mode='b'):
         sample, values = read_bytes('.test.accounts.*',
-                                    sample=80, delimiter=b'\n')
+                                    sample='80 B', delimiter=b'\n')
```
Comment (martindurant) [inline dask/bytes/tests/test_local.py]: Maybe want to add a test with different size to ensure that this is working as expected - should get different number of lines.

Comment (mrocklin) [reply dask/bytes/tests/test_local.py]: Done?

Comment (martindurant) [reply dask/bytes/tests/test_local.py]: That will do.

**Comment by martindurant** [comment]
+1 to merge this. Can press the button if no further activity by tomorrow.

**Comment by mrocklin** [comment]
Thanks for pressing the button!

**Pull request merged**